doc/index.rst

```diff
@@ -49,10 +49,12 @@ See :doc:`development` on setting up your environment and workflow.
 .. code-block:: python
 
    import fedmsg
-   fedmsg.publish(topic='testing', modname='test', msg={
+   fedmsg.publish(name='mybus', topic='testing', modname='test', msg={
```

Comment:
Sounds good, or provide both example and explain how the ``endpoint.py`` should look in both cases